### ISSUE TITLE
Suggestion of adding example

### ISSUE THREAD

**Issue by WomasL** [KNOXDEV/feather#6]
Adding a example or a video example can help new users of this API understand how to use.

**Comment by KNOXDEV** [comment]
I agree completely. I've been meaning to do this for a while, but I've been very busy lately. I'll see what I can push out within the week, along with some new API additions.

What I had in mind was, initially, to add some quick start style examples in the readme, which is in the process of being written. In addition, I also want to create a test program that will use the library to composite a complex scene, which can additionally be used as a documentation of the library's features.

Let me know if you have any questions, advice, or want to help.

**Comment by WomasL** [comment]
It's great to hear that. My personally, doesn't really understand how to use of this api. If there is a example. i will definitely invite my friends to use this api.
